### ISSUE TITLE
Exercise 3 - Your 2nd Pull Request 

### ISSUE THREAD

**Issue by CristinaRuth** [CristinaRuth/dev-together-workshop-github-basics#7]
## Your 2nd Pull Request

This exercise is to help you get familiar with the process on how to create a pull request against another user's repo.

**If you are not familiar with Git and GitHub and are completely new, please see #4 first before going through this exercise.**

In this exercise, you'll fork THIS repo and create a pull request. This mimics the process you'd be following for any Open Source contributions.

For terminologies, please refer to the presentation linked in our main `README` file.

### Steps
1. Go to https://github.com/CristinaRuth/dev-together-workshop-github-basics and at the top right, click on fork. This should create a copy of the repo under your profile. 
1. Once the copy is complete, clone the repo to your machine and make changes - create a new `“sample-your-name-or-screenname.md”` file with any text within it under the `“[Year]/[Month day]”` (e.g. 2019/Apr 9) folder. See #4 and #5 for the steps and commands.
1. Go back to your copy of the repo on github.com, go to Pull Requests and create a new pull request. Notice how the comparison is between “base repository” and “head repository”. (similar to branches) Click on “create pull request”, put some descriptive text in the next screen and click on “Create pull request”
1. Your pull request is now pending! Once one of the collaborators of this repo approves and completes it, your commit will show in the base repository https://github.com/CristinaRuth/dev-together-workshop-github-basics.

**Comment by debajit13** [comment]
I want to do this.

**Comment by CristinaRuth** [comment]
Great! Please reference this issue in your PR. We'll keep this issue open for any folks who want to tackle it. It's intended to be generic enough so that anyone can take a stab at opening their first PR :)
